specification/network/resource-manager/Microsoft.Network/2017-11-01/publicIpAddress.json

```diff
@@ -360,6 +360,13 @@
           "$ref": "#/definitions/PublicIPAddressDnsSettings",
           "description": "The FQDN of the DNS record associated with the public IP address."
         },
+        "ipTags": {
```

Comment:
Was this simply an omission in the original swagger and the service already supports this?

Reply:
The service supports and the 2017-11-01 is not shipped yet.

Reply:
@jhendrixMSFT some context:
- Not deployed yet (this is a feature branch PR). Deployment planned for tomorrow
- This is new features that made it right in time
As you imagine since I'm answering, we are working to get SDK/CLI ready for tomorrow :)